spirv_cross/src/msl.rs

```diff
@@ -49,6 +49,35 @@ pub struct ResourceBinding {
     pub force_used: bool,
 }
 
+/// A MSL shader platform.
+#[repr(u8)]
+#[derive(Copy, Clone, Debug, Hash, Eq, PartialEq)]
+pub enum Platform {
+    IOS = 0,
+    OSX = 1,
```

Comment:
nit: OS X => macOS (any casing) to match SPIRV-Cross

Reply:
Note that MSL specification uses OSX instead of macOS. I do agree that it is confusing, and "macOS" fits the other names better...